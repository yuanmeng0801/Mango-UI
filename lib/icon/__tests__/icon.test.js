import * as renderer from 'react-test-renderer'
import React from 'react'
import Icon from '../index'
import { mount } from 'enzyme'

describe('icon', () => {
  it('render successfully', () => {
    const json = renderer.create(<Icon type='labi' />).toJSON()
    expect(json).toMatchSnapshot()
  })
  it('onClick', () => {
    const component = mount(<Icon name='' />)
  })
})
