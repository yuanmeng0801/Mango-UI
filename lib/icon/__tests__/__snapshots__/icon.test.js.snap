// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon render successfully 1`] = `
<svg
  className="night-with-icon night-with-icon-6 "
>
  <use
    xlinkHref="#add"
  />
</svg>
`;
